harness/variables.go

```diff
@@ -0,0 +1,102 @@
+package harness
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	// ErrorMessageKey is the key used to retrieve or store the error message content.
+	ErrorMessageKey = "ERROR_MESSAGE"
+
+	// ErrorCodeKey is the key used to identify the specific error code associated with an error.
+	ErrorCodeKey = "ERROR_CODE"
+
+	// ErrorCategoryKey is the key used to classify the category of the error, which can help in grouping similar types of errors.
+	ErrorCategoryKey = "ERROR_CATEGORY"
+
+	// MetadataFile is the key for the file that stores metadata associated with an error, such as details about the error's source or context.
+	MetadataFile = "ERROR_METADATA_FILE"
+
+	// DroneOutputFile is the key for the file where output can exported and utilized in the subsequent steps in Harness CI pipeline.
+	DroneOutputFile = "DRONE_OUTPUT"
```

Comment:
this env "**DRONE_OUTPUT**" should be used only for CRUD of adding outputs variables . for secrets we should use **HARNESS_OUTPUT_SECRET_FILE** as env variable